### PR TITLE
Issue 6168: "redir" now mostly use OWA

### PR DESCRIPTION
**Pull request by annando**
Hopefully fixes https://github.com/friendica/friendica/issues/6168

We now switch to OWA whenever this is possibly. Additionally we ignore the network in the "redir" process, which should avoid the problems that had been mentioned in the issue.

Unrelated to this is a small AP fix for incoming posts that have got an invalid actor.

### CODE REVIEW

File: src/Model/Contact.php [hunk 1]
```diff
@@ -98,6 +98,26 @@ class Contact extends BaseObject
...
+	 */
+	public static function getBasepath($url)
+	{
+		$data = Probe::uri($url);
+		if (!empty($data['baseurl'])) {
+			return $data['baseurl'];
+		}
+
+		$urlarr = explode('/profile/', $url);
+                $basepath = $urlarr[0];
```
Comment (MrPetovan) [inline src/Model/Contact.php]: Code standards: Wrong indentation.

File: src/Model/Contact.php [hunk 2]
```diff
@@ -98,6 +98,26 @@ class Contact extends BaseObject
...
+	 * @return string basepath
+	 */
+	public static function getBasepath($url)
+	{
+		$data = Probe::uri($url);
+		if (!empty($data['baseurl'])) {
+			return $data['baseurl'];
+		}
+
+		$urlarr = explode('/profile/', $url);
```
Comment (MrPetovan) [inline src/Model/Contact.php]: Isn't this function limited to DFRN contacts?

Comment (annando) [reply src/Model/Contact.php]: Yeah.

File: src/Model/Contact.php (hunk 1 above)
Comment (annando) [reply src/Model/Contact.php]: copy&paste ;-)

**Pull request merged**